IPIP/0000-gateway-redirects.md

```diff
@@ -38,7 +38,7 @@ If a `_redirects` file exists but is unable to be processed, perhaps not even pa
 The detailed specification is added in [`http-gateways/REDIRECTS_FILE.md`](../http-gateways/REDIRECTS_FILE.md).
 
 ### Test fixtures
-QmcZzEbsNsQM6PmnvPbtDJdRAen5skkCxDRS8K7HafpAsX
+QmXqMQ5Sja34NaBEFD7x1hH4VTZWEGmmFF8fGQvAqRsJ1e
```

Comment:
@justincjohnson this CID is not provided on public DHT, and the CID in https://github.com/ipfs/kubo/pull/8890 is now different (`QmQprDnhzptPQZwC5dnSvGgmeF9yKCvnNqWA94yVAZCv3a`).

Should we update spec to `QmQprDnhzptPQZwC5dnSvGgmeF9yKCvnNqWA94yVAZCv3a`?

Reply:
Yes indeed, we should update to `QmQprDnhzptPQZwC5dnSvGgmeF9yKCvnNqWA94yVAZCv3a`.  Sorry about that.  Are you going to update that or should I?

Reply:
Updates have been pushed.